packages/@aws-cdk/aws-appsync/lib/data-source.ts

```diff
@@ -203,6 +203,25 @@ export class DynamoDbDataSource extends BackedDataSource {
   }
 }
 
+/**
+ * The authorization config in case the HTTP endpoint requires authorization
+ */
+export interface AwsIamConfig {
+  /**
+   * The authorization type required by the HTTP endpoint
+   */
+  readonly authorizationType: 'AWS_IAM';
```

Comment:
is this field necessary if it's fixed?

could we just make the interface

```ts
export interface AwsIamConfig {
  readonly signingRegion: string;
  readonly signingServiceName: string;
}
```

Reply:
So the authorizationType removes from the caller.  Building it Constructor in DataSource.